tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -4,6 +4,10 @@
  * code. Other test suites validate macros that require creating a key
  * and using it. */
 
+/* Validate that the error macros that we define in "psa/crypto_values.h"
+ * are compatible with the official PSA list. */
```

Comment:
I'm not sure if we can formally state that this is an official PSA list if it has a fix that was introduced only in this repository. Very minor fix, but according to the `C99` excerpt you mentioned, one that mentions.

Reply:
At least we're validating the numerical values, which is useful in itself.

Reply:
Sure, but that's a different point. I'm just feeling like we're just smuggling a little lie here :)

Reply:
I've now taken the exact file from [the official fix](https://github.com/ARM-software/psa-api/pull/21). It isn't an official specification yet, but I expect the header won't change again until the next patch release of the specification. So I think “the official PSA list” is now a perfectly valid claim for the macro expansions and not just the values.